test/parallel_api/ranges/stable_sort_ranges_sycl.pass.cpp

```diff
@@ -0,0 +1,60 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) 2020 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#include "support/pstl_test_config.h"
+
+#include _PSTL_TEST_HEADER(execution)
+
+#if _ONEDPL_USE_RANGES
+#include _PSTL_TEST_HEADER(ranges)
+#endif
+
+#include "support/utils.h"
+
+#include <iostream>
+
+int32_t
+main()
+{
+#if _ONEDPL_USE_RANGES
+    const int max_n = 10;
+    int data1[max_n] = {0, 1, 2, -1, 4, 5, 6, 7, 8, 9};
+    int data2[max_n] = {0, 1, 2, -1, 4, 5, -6, 7, 8, 9};
+
+    using namespace TestUtils;
+    using namespace oneapi::dpl::experimental::ranges;
+    {
+        sycl::buffer<int> A(data1, sycl::range<1>(max_n));
+        sycl::buffer<int> B(data2, sycl::range<1>(max_n));
+
+        auto exec = TestUtils::default_dpcpp_policy;
+        using Policy = decltype(TestUtils::default_dpcpp_policy);
+
+        stable_sort(exec, all_view<int, sycl::access::mode::read_write>(A));
```

Comment:
The tests for sort and stable sort are the same with the exception of suitable algos calling (two strings). Maybe, we can combine it into one test. What do you think?

Reply:
For the folder "ranges" we have one test case - one test file. It's pretty simple for statistic. 
Yes, we have some code duplication.. but the tests are so trivial.. I'm not sure that we have to combine several tests into one in these test units.